src/detect-engine-buffer.c

```diff
@@ -174,3 +174,26 @@ SigMatch *DetectBufferGetLastSigMatch(const Signature *s, const uint32_t buf_id)
     }
     return last;
 }
+
+int SCDetectSignatureAddTransform(Signature *s, int transform, void *options)
```

Comment:
Is this function out of place in this header?

Reply:
I think so : it adds a transform to the current buffer